pkg/resources/metrics.go

```diff
@@ -183,6 +183,29 @@ func CreateSmtpSecretExists(ctx context.Context, client k8sclient.Client, cr *v1
 	return v1alpha1.PhaseCompleted, nil
 }
 
+// CreateDeadMansSnitchSecretExists creates a PrometheusRule to alert if the redhat-rhoam-deadmanssnitch is present
+func CreateDeadMansSnitchSecretExists(ctx context.Context, client k8sclient.Client, cr *v1alpha1.RHMI) (v1alpha1.StatusPhase, error) {
+	installationName := InstallationNames[cr.Spec.Type]
+
+	alertName := "DeadMansSnitchSecretExists"
+	ruleName := "deadmanssnitch-secret-exists-rule"
+	alertExp := intstr.FromString(
+		fmt.Sprintf("absent(kube_secret_info{namespace='%s',secret='"+cr.Spec.NamespacePrefix+"deadmanssnitch'} == 1)", cr.Namespace),
+	)
+	alertDescription := fmt.Sprintf("The DeadMansSnitch secret has not been created in the %s namespace and may need to be created manualy", cr.Namespace)
+	labels := map[string]string{
+		"severity": "warning",
+		"product":  installationName,
+	}
+
+	ruleNs := cr.Spec.NamespacePrefix + "observability"
+	_, err := reconcilePrometheusRule(ctx, client, ruleName, ruleNs, alertName, alertDescription, sopUrlDeadMansSnitchSecretExists, alertFor10Mins, alertExp, labels)
```

Comment:
Similar question on the wait time. The alert time is set for 10 minutes, did sre have any input into this?

Reply:
Thank you. I will check with SRE

Reply:
Done. Discussed with SRE. For now, 10 min is ok